realsense_camera/src/base_nodelet.cpp

```diff
@@ -680,7 +680,15 @@ namespace realsense_camera
   void BaseNodelet::getStreamCalibData(rs_stream stream_index)
   {
     rs_intrinsics intrinsic;
-    rs_get_stream_intrinsics(rs_device_, stream_index, &intrinsic, &rs_error_);
+
+    if (stream_index == RS_STREAM_COLOR)
```

Comment:
so RS_STREAM_COLOR is secretly RS_STREAM_RECTIFIED_COLOR? is there a way to change the stream index when it's pulled out instead of masking it here?

Reply:
Technically yes, but there isn't a clean way and the value exists everywhere.